app/src/main/java/com/netmera/netmerafintech/ui/decorators/DividerItemDecorator.kt

```diff
@@ -0,0 +1,24 @@
+package com.netmera.netmerafintech.ui.decorators
```

Comment:
/*
 * Copyright (c) 2023 Netmera.
 */

Please also apply this copyright info to all of the other classes.